app/views/recipes/show.html.erb

```diff
@@ -26,6 +26,41 @@
             <h5>Instruction</h5>
             <p><%= @recipe.instruction %></p>
           </div>
+      </div>
+      <div class="missing_ingredient">
+        <h2>Missing Ingredients</h2>
+          <% recipe_ingredient_ids = @recipe_ingredients.pluck(:ingredient_id) %>
+          <% missing_ingredients = [] %>
+          <% recipe_ingredient_ids.each do |ingredient_id| %>
+            <% fridge_ingredient = FridgeIngredient.find_by(ingredient_id: ingredient_id) %>
+            <% if fridge_ingredient.nil? %>
+              <% fridge_ingredient = FridgeIngredient.new(ingredient_id: ingredient_id, quantity: 0) %>
+            <% end %>
+            <% if fridge_ingredient.quantity < @recipe_ingredients.find_by(ingredient_id: ingredient_id).quantity %>
+              <% missing_ingredients << Ingredient.find(ingredient_id) %>
+            <% end %>
+          <% end %>
+          <% if missing_ingredients.empty? %>
+            <h3>You have everything for your recipe!</h3>
+            <p>If you want to make your recipe now, we can automatically remove the ingredietn fromyour fridgi by clicking the link below!</p>
```

Comment:
I now, tiny stuff, but please fix spelling errors

(remove the ingredient from you Fridgi by clicking the link below!)

Reply:
Important details